### PR TITLE
PPTP-2945: live services fixes

### PR DESCRIPTION
**Pull request by mauro-fanton**

### CODE REVIEW

File: app/assets/stylesheets/_returns.scss [hunk 1]
```diff
@@ -11,60 +11,3 @@ th.govuk-table__header.row-border {
 td.govuk-table__cell.row-border {
   border-bottom: 1px solid #b1b4b6 !important;
 }
-
-.hmrc-user-research-banner {
```
Comment (mauro-fanton) [inline app/assets/stylesheets/_returns.scss:15]: This apply the same fixes of Pan to the search banner

File: conf/app.routes
Comment (mauro-fanton) [inline conf/app.routes:1]: not sure if these changes are what live services meant when saying reformat app.route.

File: app/assets/stylesheets/_returns.scss (hunk 1 above)
Comment (liampan) [reply app/assets/stylesheets/_returns.scss:15]: Do we need this file at all? 
It feels weird to have returns.css sheet in reg...

File: conf/app.routes
Comment (liampan) [reply conf/app.routes:1]: Yep I think so

File: app/assets/stylesheets/_returns.scss (hunk 1 above)
Comment (mauro-fanton) [reply app/assets/stylesheets/_returns.scss:15]: I am nervous to remove the css completely as I am not sure if these are used by imported components. What I can do to be safe is to move what is in _returns.scss to _registration.scss and remove the _return.scss file, as I see your point Pan.

Comment (liampan) [reply app/assets/stylesheets/_returns.scss:15]: Yeah sounds like a better idea than leaving this one here, custom CSS is to be avoided as much as possible tbh

Comment (mauro-fanton) [reply app/assets/stylesheets/_returns.scss]: done

**Review by liampan — APPROVED** [review]

**Pull request merged**